test/gtest/uct/test_md.cc

```diff
@@ -638,16 +644,28 @@ UCS_TEST_SKIP_COND_P(test_md, reg_advise,
                             &memh);
     ASSERT_UCS_OK(status);
     ASSERT_TRUE(memh != UCT_MEM_HANDLE_NULL);
+    if (check_non_blocking) {
+        vmpin_after = ucs::get_proc_self_status_field("VmPin");
+        ASSERT_EQ(vmpin_before, vmpin_after);
+    }
 
-    status = uct_md_mem_advise(md(), memh, (char *)address + 7,
-                               32 * UCS_KBYTE, UCT_MADV_WILLNEED);
-    EXPECT_UCS_OK(status);
+    if (advise_size) {
+        status = uct_md_mem_advise(md(), memh, (char*)address + advice_offset,
+                                   advise_size, UCT_MADV_WILLNEED);
+        EXPECT_UCS_OK(status);
+    }
 
     status = uct_md_mem_dereg(md(), memh);
     EXPECT_UCS_OK(status);
     free(address);
 }
 
+UCS_TEST_SKIP_COND_P(test_md, reg_advise,
```

Comment:
why need this test now? It can be a part of test_md_non_blocking

Reply:
I just thought that it can make sense to leave it as is and test nonblock flag with the default configuration.